server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/JmapRfc8621Configuration.scala

```diff
@@ -22,24 +22,28 @@ package org.apache.james.jmap.model
 import java.net.URL
 
 import org.apache.commons.configuration2.Configuration
-import org.apache.james.jmap.routes.UploadRoutes
-import org.apache.james.jmap.routes.UploadRoutes.Size
 import org.apache.james.util
 import org.apache.james.util.Size
 
+import scala.util.{Failure, Success, Try}
+
 
 object JmapRfc8621Configuration {
   val LOCALHOST_URL_PREFIX: String = "http://localhost"
   private def from(urlPrefixString: String): JmapRfc8621Configuration = JmapRfc8621Configuration(urlPrefixString)
   var LOCALHOST_CONFIGURATION: JmapRfc8621Configuration = from(LOCALHOST_URL_PREFIX)
   val URL_PREFIX_PROPERTIES: String = "url.prefix"
 
-  val UPLOAD_LIMIT_PROPERTIES: String = "upload.max.mb.size"
-  val UPLOAD_LIMIT_30_MB: String = "30M"
+  val UPLOAD_LIMIT_PROPERTIES: String = "upload.max.size"
+  val UPLOAD_LIMIT_30_MB: Long = 30 * 1024 * 1024
   var MAXSIZE_UPLOAD: Option[util.Size] = None
 
   def from(configuration: Configuration): JmapRfc8621Configuration = {
-    MAXSIZE_UPLOAD = Some(Size.parse(configuration.getString(UPLOAD_LIMIT_PROPERTIES, UPLOAD_LIMIT_30_MB)))
+    MAXSIZE_UPLOAD = Try(Size.parse(configuration.getString(UPLOAD_LIMIT_PROPERTIES))) match {
+      case Success(size: Size) => Some(size)
+      case Failure(_) => None
```

Comment:
Please propagate the exception. We should reject invalid configuration.

Reply:
Well it rejected invalid configuration and use default value instead.